react-native-gutenberg-bridge/android/src/main/java/org/wordpress/mobile/WPAndroidGlue/WPAndroidGlueCode.java

```diff
@@ -88,6 +89,10 @@ public boolean isContentChanged() {
         void onQueryCurrentProgressForUploadingMedia();
     }
 
+    public interface OnEditorMountListener {
+        void onEditorDidMount(Boolean hasUnsupportedBlocks);
```

Comment:
Let's replace `Boolean` with `boolean` here (it's a different class 😬) since the latter is what the interface is expecting.

Reply:
Done in eb066b6.